### DISCUSSION TITLE
is it possible to use the Auth0 React SDK with the Web3Auth SDK?

### DISCUSSION THREAD

**Discussion by RoniFinTech** [Web3Auth/Web3Auth#901] [Help]
Auth0 React has a useful component <Auth0Provider/> and I wonder if it's possible to use something similar when integrating Web3Auth and Auth0.

Thanks,
Roni

**Comment by shahbaz17** [comment]
Hey @RoniFinTech 

Does `@auth0/auth0-react` return JWT(`id_token`) upon successful authentication?

If yes, then you can use it with Web3Auth by creating a custom verifier and passing the JWT to web3auth:

```js
import { WALLET_ADAPTERS, CHAIN_NAMESPACES } from "@web3auth/base";

await web3auth.connectTo(WALLET_ADAPTERS.OPENLOGIN, {
  loginProvider: "jwt",
  extraLoginOptions: {
    id_token: idToken, // returned by Auth0 from `@auth0/auth0-react`
    verifierIdField: "sub", // same as your JWT Verifier ID
  },
});
```
